cloud/ec2_util.go

```diff
@@ -215,6 +223,129 @@ func timeTilNextHourlyPayment(host *host.Host) time.Duration {
 	return nextPaymentTime.Sub(now)
 }
 
+func getEc2SSHOptions(h *host.Host, keyName string) ([]string, error) {
```

Comment:
there's something very much like this in the host package.

Reply:
removed this function and now using the function from the host package.